src/app/utils.ts

```diff
@@ -130,4 +144,21 @@ export default class Utils {
     return `rgb(${red},${green},${blue})`;
   }
 
+  /**
+   * Find difference between objects
+   * @param currentObj
+   * @param baseObj 
+   * @returns difference 
+   */
+  public static difference(currentObj, baseObj) {
```

Comment:
We cannot use lodash inbuilt?

Reply:
There is no direct function to get the objects difference, this method is built using the lodash functions `isEqual, transform, isObject`.